NativeUI/Elements/NativeRectangle.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Drawing;
+using GTA.Native;
+
+namespace NativeUI.Elements
+{
+    public class NativeRectangle : NativeElement
+    {
+        #region Constructors
+
+        public NativeRectangle(Point position, Size size) : this(position, size, Color.White) { }
+
+        public NativeRectangle(Point position, Size size, Color color)
+        {
+            Size = size;
+            Position = position;
+            Color = color;
+        }
+
+        #endregion
+
+        #region Drawing
+
+        /// <summary>
+        /// Draws the rectangle on screen.
+        /// </summary>
+        public override void Draw()
```

Comment:
Enabled property is ignored. Also this class should override Draw(Size offset). Or maybe leave it since it'd correctly throw NotImplemented. Up to you.